platform/cavium/rules.mk

```diff
@@ -1,21 +1,13 @@
-CAVM_LIBSAI = libsai.deb
-$(CAVM_LIBSAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
-CAVM_SAI = sai.deb
-$(CAVM_SAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
-XP_TOOLS = xp-tools.deb
-$(XP_TOOLS)_PATH = $(PLATFORM_PATH)/cavm_sdk
-XPSHELL = xpshell.deb
-$(XPSHELL)_PATH = $(PLATFORM_PATH)/cavm_sdk
+include $(PLATFORM_PATH)/cavm-sai.mk
+include $(PLATFORM_PATH)/docker-syncd-cavm.mk
+include $(PLATFORM_PATH)/docker-orchagent-cavm.mk
 
-SONIC_COPY_DEBS += $(CAVM_LIBSAI) $(CAVM_SAI) $(XP_TOOLS) $(XPSHELL)
+SONIC_ALL += $(DOCKER_SYNCD_CAVM) \
+	     $(DOCKER_ORCHAGENT_CAVM)
```

Comment:
You need this one
`include $(PLATFORM_GENERIC_PATH)/rules.mk`
see https://github.com/Azure/sonic-buildimage/commit/e3b126893ba37f0fe3292f9f953c4d9a63d4f631